src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -155,6 +158,18 @@ public boolean isFake()
         return fake;
     }
 
+    @Nonnull
+    @Override
+    public EnumSet<UserFlag> getFlags()
+    {
+        return flagsSet;
```

Comment:
You're returning a modifiable set here. This shouldn't be a problem if you would compute it inplace rather than ahead of time.

Reply:
Was changed to `UserFlag.getFlags(flags)`